management-api-server/src/test/java/com/datastax/mgmtapi/BaseDockerIntegrationTest.java

```diff
@@ -167,6 +167,15 @@ protected ArrayList<String> getEnvironmentVars() {
         "DSE_MGMT_EXPLICIT_START=true");
   }
 
+  protected String getUser() {
+    // The default should be either "cassandra:root" or "dse:root"
+    // Subclasses can override this to test alternate user behavior
+    if (this.version.startsWith("dse")) {
```

Comment:
"dse:root" ?

The DSE test fails now.

Reply:
The DSE tests are passing in this PR now. I also loaded the image into my Kind cluster for cass-operator and ran the tests against that and it is working for me.